Add android 11 support for email intent params/targets

pprattis opened the pull request.

jfoclpf commented:
Hi @pprattis thank you very much for the PR and apologies the late reply

Does this PR is still applicable? What exactly does it solve? Can you be more specific, for the record

Thanks again

pprattis commented:
Hi @jfoclpf , this PR is still applicable, let me explain my fixes.
1) I added queries element in plugin.xml, starting from Android 11 androidmanifest is required to declare which apps or components the app intends to interact with. I added the SENDTO action to send emails.
2) I adjusted the getDraft method, so that the chooser allows multiple email clients to be included as options, with the first app as the default in the chooser dialog. This change is really important so that the app can detect and interact with email clients on Android 11+.
